bot/exts/utilities/githubinfo.py

```diff
@@ -76,7 +79,22 @@ class GithubInfo(commands.Cog):
 
     def __init__(self, bot: Bot):
         self.bot = bot
-        self.repos = []
+        self.pydis_repos: dict = {}
+
+    async def cog_load(self) -> None:
+        """Function to be run at cog load."""
+        self.refresh_repos.start()
+
+        self.stored_repos_json = Path(__file__).parent.parent.parent / "resources" / "utilities" / "stored_repos.json"
```

Comment:
This doesn't need to be a self attribute. This can be a constant defined at the top of the file: 
```py
STORED_REPO_PATH = Path("bot/resources/utilities/store_repos.json")
```